splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -222,13 +225,15 @@ def install_libraries(
         sys.exit(1)
 
     deps_resolver = "--use-deprecated=legacy-resolver " if pip_legacy_resolver else ""
+    custom_flag = pip_custom_flag if pip_custom_flag else ""
     pip_install_command = (
         f'-r "{requirements_file_path}" '
         f"--no-compile "
```

Comment:
The ticket did not specify it explicitly, should we consider `--no-compile --prefer-binary --ignore-installed` as a default for `--pip-custom-flag`?

Reply:
I think we can. So if the user does not specify `--pip-custom-flag`, we will add these three to the command, but if the user adds custom flags then `--no-compile --prefer-binary --ignore-installed` will be missing from the command . If the user still wants them, he will have to include them in the added custom flags

Reply:
Yes, let's do it like this.

Additionally regarding OS-dependent libraries - we can skip updating them in a context of this change.

Reply:
updated